content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +296,219 @@ clicked, sort the data in different ways.
+   :alt: The icons in the top right corner than can be clicked to present the information in
          different ways.
 
 List view
 ~~~~~~~~~
 
-The default view of the services is the list view. This presents all the services in chronological
-order, from oldest to newest, in a list.
+The default view of the service records is a list view. This presents all the services, first
+grouped alphabetically by type of service, then grouped by status.
+
+The information can be re-sorted by any column. At the top of each column, hover over the
+:guilabel:`column name`, and an arrow appears in the far-right of that column. Click the arrow to
+sort the data by that specific column.
+
+The default sorting is descending alphabetical order (A to Z), represented by a :guilabel:`⌄ (down
+arrow)`. Click the :guilabel:`⌄ (down arrow)` to reverse the alphabetical order (Z to A). The
+:guilabel:`⌄ (down arrow)` changes to an :guilabel:`^ (up arrow)`.
 
-The information can be sorted by any column. At the top of each column, hover over the column name.
-An arrow appears to the right of the name. Click the arrow to sort the data by that specific column.
+The two exceptions to this sorting is the default :guilabel:`Date` column and the :guilabel:`Cost`
+column. The :guilabel:`Date` column sorts the information in chronological order (January to
+December) instead of alphabetical order, while the :guilabel:`Cost` column sorts the information by
+repair price, from lowest to highest.
 
-The default sorting is descending alphabetical order (A to Z). Click the arrow again to reverse the
-order, and go in reverse alphabetical order (Z to A). The one exception to this sorting is the
-default :guilabel:`Date` column, which sorts the information in chronological order (January to
-December) instead of alphabetical order.
+Add a service
+*************
+
+To add a service record from the list view, click the :guilabel:`New` button, and a service form
+loads. :ref:`Enter all the information <service/service-form>` on the service form.
+
+The form automatically saves as data is entered.
 
 Kanban view
 ~~~~~~~~~~~
 
 To view services by their stage, click on the :guilabel:`Kanban icon`, which is the second icon in
-the top right, and appears as four black squares in a cube.
+the top-right corner, and appears as two different length bars beneath a line.
+
+All services with the same status appear in the corresponding Kanban column, from
+:guilabel:`Running` to :guilabel:`Cancelled`.
+
+The number of repairs for each type of service appear in the far-right of each Kanban column header.
+
+The status of any scheduled activities appear in the color-coded bar beneath each Kanban column
+title. Repairs with activities scheduled in the future appear green, activities due today appear
+yellow, overdue activities appear red, and repairs with no activities scheduled appear gray.
+
+Each Kanban card displays a color-coded :guilabel:`activity icon`, indicating what type of activity
```

Comment:
I'd use a clock icon here, as we don't actually see "activity icon" literally, on the UI. If you want to describe an icon, how about `(activity icon)`, in parentheses?

Reply:
The only issue is, it is only a clock if there are no activities scheduled. if there are, the icon changes accordingly- a phone for a phone call, 3-line like a written entry for a meeting, etc. But I agree- since it's not the actual name, I'll put it in parenthesis. And I added a line with an explanation for a phone call icon, to make it more clear that these icons are different.